.circleci/dynamic/index.ts

```diff
@@ -0,0 +1,108 @@
+import * as fs from "fs";
```

Comment:
This use of the TypeScript Config SDK should produce the same output as the previous [config.yml](https://github.com/getlocalci/circleci-tutorial-config-sdk/pull/2/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47)